### ISSUE TITLE
linter-csslint:: Error while running CSSLint!

### ISSUE THREAD

**Issue by riccardo24-7** [AtomLinter/linter-csslint#293]
Hi guys! I have the problem, when I have saved my css file
![Screenshot_1](https://user-images.githubusercontent.com/60128039/73021338-6c0b6c00-3e38-11ea-9948-f9687930393a.png)
I installed the package linter-csslint 2.0.6. I'm using Atom 1.43.0 x64

**Comment by Arcanemagus** [comment]
Do you have `csslint` installed on your system? You should be able to run that from a terminal interface without needing to specify anything else, and without errors.

**Comment by riccardo24-7** [comment]
Yes, csslint installed on my PC. This problem I saw, when I was saving my file with extension .css. But fortunately the website performed correctly

**Comment by riccardo24-7** [comment]
> Do you have `csslint` installed on your system? You should be able to run that from a terminal interface without needing to specify anything else, and without errors.


Yes, csslint installed on my PC. This problem I saw, when I was saving my file with extension .css. But fortunately the website performed correctly

**Comment by FERNII** [comment]
Line  linter-csslint:: Error while running CSSLint! 
I have this error

"node" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.

**Comment by uly94** [comment]
I have this annoying error, too. How to solve it? I'd prefer to not remove ccslint.
![csslint-error](https://user-images.githubusercontent.com/18552295/78650180-d885fb00-78be-11ea-9b28-b6e6f622caac.PNG)

**Comment by riccardo24-7** [comment]
i founded the solution, the solution but it's very simple and maybe stupid. I deleted this package and now i don't have this problem

**Comment by riccardo24-7** [comment]
only this solution i founded, unfortunately

**Comment by uly94** [comment]
> i founded the solution, the solution but it's very simple and maybe stupid. I deleted this package and now i don't have this problem

Yeah, but I hoped not to remove the package to have a solution

**Comment by riccardo24-7** [comment]
> > i founded the solution, the solution but it's very simple and maybe stupid. I deleted this package and now i don't have this problem
> 
> Yeah, but I hoped not to remove the package to have a solution

Ok, maybe tomorrow i'll try to find another one solution

**Comment by FERNII** [comment]
Thanks I deleted the package too.

**Comment by Auntie-Jen** [comment]
I am also getting this error message.  Funny thing is, I just started getting it.  I am in a Udemy class, and added this package for the class.  Is the only solution to uninstall?  At the same time this is showing, it says that my linter is not seeing any errors in my code, and they appear to be rendering correctly in page, and also in developer tools.  Any help would be appreciated, and thanks in advance.

**Comment by AdityaAgrawal-03** [comment]
I am also having an issue in this **linter-csslint** package.
![Screenshot (18)](https://user-images.githubusercontent.com/55623652/82684366-1624c480-9c70-11ea-8a4f-556e6e3cc90f.png)

Can anyone help??

**Comment by SeekerLorca** [comment]
<img width="504" alt="Screen Shot 2020-05-25 at 1 27 56 AM" src="https://user-images.githubusercontent.com/30909700/82794280-11564100-9e27-11ea-99ca-7afa15a49368.png">
Same issue here...

**Comment by malvengarlix** [comment]
shit I got the same thing.

**Comment by negru13** [comment]
Install Node.js to get rid of that error : https://nodejs.org/en/

**Comment by LhSalerno** [comment]
alguem conseguiu se livrar desse erro ?
ele ta roubando meu juizo. não estou conseguin do rodar o Css no meu HTML

**Comment by sandesh-saidapur** [comment]
> I am also having an issue in this **linter-csslint** package.
> ![Screenshot (18)](https://user-images.githubusercontent.com/55623652/82684366-1624c480-9c70-11ea-8a4f-556e6e3cc90f.png)
> 
> Can anyone help??

bro did you find the solution to it??im facing the same exact problem but applied css performs correctly on the website

**Comment by nahidsultana28** [comment]
> I am also having an issue in this **linter-csslint** package. ![Screenshot (18)](https://user-images.githubusercontent.com/55623652/82684366-1624c480-9c70-11ea-8a4f-556e6e3cc90f.png)
> 
> Can anyone help??

Did you find the solution for this??

**Comment by nahidsultana28** [comment]
i had the same problem, i removed the csslint package, but i am getting IDENT errror in atom editor.. how do i get rid of that??

**Comment by rameennaz** [comment]
I had the same problem anyone solve this for me?????

**Comment by Saifulhuq01** [comment]
guys I had to fixed that issue its very easy just uninstall a css-lint and linter package after its done install node.js recommended version afterthat reinstall that linter and css-lint packages its working without error

**Comment by Gastrebucq** [comment]
> Hi guys! I have the problem, when I have saved my css file ![Screenshot_1](https://user-images.githubusercontent.com/60128039/73021338-6c0b6c00-3e38-11ea-9948-f9687930393a.png) I installed the package linter-csslint 2.0.6. I'm using Atom 1.43.0 x64

I found it! (someone told me, actually)
You have to install node.js in your computer and that will let you use the package :)
